polls/views.py

```diff
@@ -109,7 +109,10 @@ def get_actions(self):
         actions = {}
 
         if can_create_question(self.request):
-            actions['create'] = Action(method='POST', attributes=('question', 'choices'))
+            actions['create'] = Action(method='POST', attributes=(
+                Attribute(name='question', category='text'),
+                Attribute(name='choices', category='array[text]'),
```

Comment:
Nip: The name is usually used as a label. Perhaps capitalize it may make sense. But for now, a css property will do the job for me.

Thanks.